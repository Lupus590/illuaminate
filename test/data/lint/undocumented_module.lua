--- @module f

return 0

-- config: (lint (only doc:undocumented))
