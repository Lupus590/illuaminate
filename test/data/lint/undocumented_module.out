undocumented_module.lua:[3:8-3:8]: Module is missing documentation [doc:undocumented]
   │
 3 │ return 0
   │        ^
No errors and 1 warnings
